packages/modules/src/modules/epics/ContributionsEpic.tsx

```diff
@@ -19,6 +19,7 @@ import { HasBeenSeen, useHasBeenSeen } from '../../hooks/useHasBeenSeen';
 import { isProd } from '../shared/helpers/stage';
 import { withParsedProps } from '../shared/ModuleWrapper';
 import { ChoiceCardSelection, ContributionsEpicChoiceCards } from './ContributionsEpicChoiceCards';
+import { countryCodeToCountryGroupId } from '@sdc/shared/dist/lib';
```

Comment:
this should be `@sdc/shared/lib` (vscode also auto adds them incorrectly which is annoying)

Reply:
✅